L1Trigger/TrackFindingTracklet/python/Demonstrator_cfi.py

```diff
@@ -3,9 +3,12 @@
 
 TrackTriggerDemonstrator_params = cms.PSet (
 
-  LabelIn  = cms.string( "TrackFindingTrackletProducerDRin"    ), #
+  LabelIn  = cms.string( "TrackFindingTrackletProducerTM"    ), #
   LabelOut = cms.string( "TrackFindingTrackletProducerDR"      ), #
-  DirIPBB  = cms.string( "/heplnw039/tschuh/work/proj/DRinDR/" ), # path to ipbb proj area
-  RunTime  = cms.double( 4.5 )                                    # runtime in us
+  DirIPBB  = cms.string( "/heplnw039/tschuh/work/proj/tmdr/" ), # path to ipbb proj area
```

Comment:
Please add comment making clear that these parameters are for ModelSim runs of FW.

Reply:
done